source/batch-mode/batch-read.txt

```diff
@@ -57,6 +57,77 @@ Schema Inference
 
          .. include:: /scala/schema-inference.rst
 
+.. _spark-schema-hint:
+
+Specify Known Fields with Schema Hints
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can specify a schema containing known field values to use during
+schema inference by specifying the ``schemaHint`` configuration option. You can
+specify the ``schemaHint`` option in any of the following Spark formats:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 35 65
+
+   * - Type
+     - Format
+
+   * - DDL
+     - ``<field one name> <FIELD ONE TYPE>, <field two name> <FIELD TWO TYPE>``
+
+   * - SQL DDL
+     - ``STRUCT<<field one name>: <FIELD ONE TYPE>, <field two name>: <FIELD TWO TYPE>``
+
+   * - JSON
+     - .. code-block:: json
+          :copyable: false
+           
+          { "type": "struct", "fields": [
+          { "name": "<field name>", "type": "<field type>", "nullable": <true/false> },
+          { "name": "<field name>", "type": "<field type>", "nullable": <true/false> }]}
+
+The following example shows how to specify the ``schemaHint`` option in each
+format by using the Spark shell. The example specifies a ``string`` field named
+``"value"`` and an ``integer`` field named ``"count"``.
```

Comment:
S: in this context, I am not sure it's necessary to monospace string and integer. What about:
```suggestion
format by using the Spark shell. The example specifies a string-valued field named
``"value"`` and an integer-valued field named ``"count"``.
```